tests/unit/db/services/test_owid_charts_service.py

```
@@ -20,6 +20,10 @@
 )
 
 
+def delete_chart(id: int) -> bool:
```

Ruff (0.15.21)

[error] Function argument `id` is shadowing a Python builtin

(A002)